extjsdk/src/main/java/io/vantiq/extjsdk/ExtensionServiceMessage.java

```diff
@@ -25,8 +25,8 @@
     public static final String OP_QUERY = "query";
     public static final String OP_RECONNECT_REQUIRED = "reconnectRequired";
     public static final MediaType CONTENT_TYPE_JSON = MediaType.parse("application/json; charset=utf-8");
-    public static final String RETURN_HEADER = "REPLY_ADDR_HEADER";
-    public static final String REPLY_ADDRESS = "X-Reply-Address";
+    public static final String ORIGIN_ADDRESS_LOCATION = "REPLY_ADDR_HEADER";
+    public static final String RESPONSE_ADDRESS_LOCATION = "X-Reply-Address";
```

Comment:
I felt the name change was necessary because every time I thought of them I couldn't figure out which was in the query message and which was in the response. The origin/response is intended to signify whether it is the original message or the response. The address_location is saying that it is the location in the map of the address that will be used.

Reply:
Maybe call them ...ADDRESS_HEADER then.